src/components/ThreeDEditor.jsx

```diff
@@ -714,11 +714,11 @@ export class ThreeDEditor extends React.Component {
 
     render() {
         return (
-            <StyledEngineProvider injectFirst>
-                <ThemeProvider theme={DarkMaterialUITheme}>
+            <ThemeProvider theme={DarkMaterialUITheme}>
+                <ScopedCssBaseline enableColorScheme>
```

Comment:
This I suppose is to limit scope for additional css (fullscreen for three.js editor)